types/src/traits/election.rs

```diff
@@ -234,6 +269,83 @@ pub trait Election<TYPES: NodeType>: Clone + Eq + PartialEq + Send + Sync + 'sta
         is_valid_signature && is_valid_vote_token
     }
 
+    /// Accumlate the vote, return the QC if the threshold was reached, if not return the updated accumlator
+    #[allow(clippy::type_complexity)]
+    fn accumulate_qc_vote(
+        &self,
+        encoded_key: &EncodedPublicKey,
+        encoded_signature: &EncodedSignature,
+        leaf_commitment: Commitment<Self::LeafType>,
+        vote_token: TYPES::VoteTokenType,
+        view_number: TYPES::Time,
+        accumlator: CertificateAccumulator<TYPES::VoteTokenType, Self::LeafType>,
+    ) -> Either<
+        CertificateAccumulator<TYPES::VoteTokenType, Self::LeafType>,
+        QuorumCertificate<TYPES, Self::LeafType>,
+    > {
+        if !self.is_valid_vote(
+            encoded_key,
+            encoded_signature,
+            VoteData::Yes(leaf_commitment),
+            view_number,
+            // Ignoring deserialization errors below since we are getting rid of it soon
```

Comment:
I think this was copied over from before, but I'm not sure what exactly we're getting rid of. Serialiazation? Does anyone know? If not we can probably delete the comment and the one below

Reply:
Maybe the code below this comment used to do some deserialization, and we forgot to remove/update the comment? It doesn't seem to be related to serialization now. I guess we can delete it.